cristin-import/src/main/resources/reference.conf

```diff
@@ -0,0 +1,5 @@
+doi {
```

Comment:
This is the default configuration that enables the ONLINE validation by default. It is overriden in tests by a similar file.